app/__init__.py

```diff
@@ -350,17 +370,38 @@ def send_after_event_mail():
                     send_after_event(organizer.user.email, event.id,
                                      upcoming_events)
 
-#logging.basicConfig()
+
+# logging.basicConfig()
 sched = BackgroundScheduler(timezone=utc)
 sched.add_job(send_after_event_mail,
               'cron',
               day_of_week='mon-fri',
               hour=5,
               minute=30)
 
-#sched.start()
+
+# sched.start()
+
+def send_mail_to_expired_orders():
+    with app.app_context():
+        orders = DataGetter.get_expired_orders()
+        for order in orders:
+            print order.event.name
+            print 'HELLLO'
+            send_email_for_expired_orders('adityavyas17@gmail.com', order.event.name, order.get_invoice_number(),
+                                          url_for('ticketing.view_order_after_payment',
+                                                  order_identifier=order.identifier, _external=True))
 
 
+#logging.basicConfig()
+order_sched = BackgroundScheduler(timezone=utc)
+order_sched.add_job(send_mail_to_expired_orders,
+                    'interval',
+                    seconds=10)
```

Comment:
Every 10 seconds is too less of an interval for this job ... make it 30 mins or atleast an hour ... We don't want the user to get an email as soon as he leaves the site ... If he gets an email after a while, there's more chance of him/her rethinking and changing their mind